src/components/wizard/wizard.component.ts

```diff
@@ -152,6 +152,7 @@ export class WizardComponent implements OnInit, OnDestroy {
     constructor(protected _wizardService: WizardService<WizardStepComponent>) {
         // watch for changes to valid subject
         this._wizardService.valid$.pipe(
+            takeUntil(this._onDestroy),
```

Comment:
`takeUntil` should always be the last operator (unless there is a `shareReplay` operator which should come after)